### PR TITLE
serviced support #8492

### PR DESCRIPTION
**Pull request by anatol-sialitski**

### CODE REVIEW

**Comment by rymsha** [comment]
@gbbirkisson any suggestions to improve this?

File: modules/runtime/src/service/systemd/xp.service [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+Wants=network-online.target
+After=network-online.target
+
+[Service]
+Type=simple
+PrivateTmp=true
+Environment=XP_INSTALL=/opt/enonic/xp
+Environment=XP_JAVA_HOME=${XP_INSTALL}/jdk
+
+WorkingDirectory=/opt/enonic/xp
```
Comment (rymsha) [inline modules/runtime/src/service/systemd/xp.service:13]: `/opt` Is it commonly used directory nowadays?

File: modules/runtime/src/service/systemd/xp.service [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+LimitFSIZE=infinity
+
+KillSignal=SIGTERM
+
+KillMode=process
+
+SendSIGKILL=no
+
+SuccessExitStatus=143
```
Comment (rymsha) [inline modules/runtime/src/service/systemd/xp.service:37]: does this apply to sh script? we don't directly call java, but does script propagate same java exit code (143 means java exited normally after SIGTERM)

File: modules/runtime/src/service/systemd/xp.service (hunk 1 above)
Comment (gbbirkisson) [reply modules/runtime/src/service/systemd/xp.service:13]:
```
/opt – Optional Packages

The /opt directory contains subdirectories for optional software packages. It’s commonly used by proprietary software that doesn’t obey the standard file system hierarchy – for example, a proprietary program might dump its files in /opt/application when you install it.
```

So if the XP distro is installed there this makes sense.

File: modules/runtime/src/service/systemd/xp.service (hunk 2 above)
Comment (rymsha) [reply modules/runtime/src/service/systemd/xp.service:37]: @anatol-sialitski can you verify this, please

Comment (gbbirkisson) [reply modules/runtime/src/service/systemd/xp.service:37]: Doing a test on my machine:

```console
$XP_HOME=$HOME/.enonic/sandboxes/jwt/home ./bin/server.sh

....

CTRL+C

$echo $?
130
```

Comment (gbbirkisson) [reply modules/runtime/src/service/systemd/xp.service:37]: Killing with sigterm results in 143.

**Review by rymsha — APPROVED** [review]

**Pull request merged**